samples/msal-browser-samples/TypescriptTestApp2.0/src/AuthModule.ts

```diff
@@ -1,6 +1,6 @@
 import { PublicClientApplication, SilentRequest, AuthenticationResult, Configuration, LogLevel, AccountInfo, InteractionRequiredAuthError, RedirectRequest, PopupRequest, EndSessionRequest } from "@azure/msal-browser";
 import { UIManager } from "./UIManager";
-import { SsoSilentRequest } from "@azure/msal-browser/dist/src/request/SsoSilentRequest";
+import { SsoSilentRequest } from "@azure/msal-browser/dist/request/SsoSilentRequest";
```

Comment:
wouldn't it be cleaner to import `SsoSilentRequest `along with the rest above (ln 1)?

Reply:
Agreed, should be imported from the main export on line 1.

Reply:
@derisen @jasonnutter fair shout! I had blinders on while looking at the file. Addressed in the latest changes.